### PR TITLE
Update ipyvtklink docs

### LINKED ISSUE #2050: Update documentation for using ipyvtklink
The documentation at https://docs.pyvista.org/user-guide/jupyter/ipyvtk_plotting.html#installation is slightly wrong. 
To use ipyvtklink backend, itkwidgets package is not required. 

The following combination of packages worked for me on jupyterlab ( v 2.x) - 

PIP packages

ipyvtklink==0.2.2
ipycanvas==0.10.2
ipyevents==2.0.1

Jupyter lab extension packages

ipycanvas@0.10.2
ipyevents@2.0.1

-----

**System Information:**

```txt
--------------------------------------------------------------------------------
  Date: Thu Jan 20 19:59:55 2022 UTC

                OS : Linux
            CPU(s) : 8
           Machine : x86_64
      Architecture : 64bit
               RAM : 54.9 GiB
       Environment : Jupyter
        GPU Vendor : VMware, Inc.
      GPU Renderer : llvmpipe (LLVM 10.0.0, 256 bits)
       GPU Version : 3.3 (Core Profile) Mesa 20.0.8

  Python 3.8.0 (default, Feb 25 2021, 22:10:10)  [GCC 8.4.0]

           pyvista : 0.33.2
               vtk : 9.0.1
             numpy : 1.19.5
           imageio : 2.9.0
           appdirs : 1.4.4
            scooby : 0.5.7
        matplotlib : 3.3.4
           IPython : 7.16.1
        ipyvtklink : 0.2.1
             scipy : 1.4.1
              tqdm : 4.60.0
            meshio : 4.4.1
--------------------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by banesullivan**
Resolves #2050 to update the documentation for using `ipyvtklink`

Changes:

- Do not pin versions of packages
- Do not include itkwidgets
- Update notes about JupyterLab 3 as ipyvtklinks dependencies handle it now
- Update to use `set_jupyter_backend` in example
- Remove `labextension` commands
- Explicitly require JupyterLab `>=3`

### CODE REVIEW

**Comment by banesullivan** [comment]
@saswatac, would you be able to review this for us? 😄

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (saswatac) [inline doc/user-guide/jupyter/ipyvtk_plotting.rst:66]: unless I am mistaken, by default pyvista does not include ipyvtklink. So we should add it here.

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (saswatac) [inline doc/user-guide/jupyter/ipyvtk_plotting.rst:66]: same comment as above.

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (saswatac) [inline doc/user-guide/jupyter/ipyvtk_plotting.rst:48]: to be on the safe side, I would also pin the conda/pypi packages of ipycanvas and ipyevents at the same version.

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (banesullivan) [reply doc/user-guide/jupyter/ipyvtk_plotting.rst:66]: Good catch 🤦🏻‍♂️

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (banesullivan) [reply doc/user-guide/jupyter/ipyvtk_plotting.rst]: I went the other direction and un-pinned these. That way user's are always installing the latest versions

Comment (banesullivan) [reply doc/user-guide/jupyter/ipyvtk_plotting.rst]: These were pinned during the transition to JupyterLab 3 where a `ipyvtklink`'s dependencies weren't ready yet. But now everything should be working well

Comment (banesullivan) [reply doc/user-guide/jupyter/ipyvtk_plotting.rst]: Actually, I removed these `jupyter labextension *` commands because they aren't needed anymore with JupyterLab 3

**Review by akaszynski — APPROVED** [review]
Approving as I see no issue with this PR. @banesullivan, it's up to you if you want to wait for @saswatac's approval.

File: doc/user-guide/jupyter/ipyvtk_plotting.rst
Comment (saswatac) [reply doc/user-guide/jupyter/ipyvtk_plotting.rst]: ok, sounds good to me.

**Pull request merged**
